src/main/resources/db/changelog/20220329154201-owner-product-key-change.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20220329154201-1" author="wpoteat">
```

Comment:
Each logical step of this should be done as a separate change set to make error handling and reverts much easier. It also clarifies which pieces are DB-specific with specific changesets there. PostgreSQL, for instance, has no need to do the shenanigans with the primary/foreign keys, and shouldn't be subject to any goofy rules MySQL has.